### ISSUE TITLE
Several buttons are not labeled properly

### ISSUE THREAD

**Issue by PatchMixolydic** [Darkempire78/OpenCalc#405]
**Describe the bug**
The ellipses menu, more buttons button, and exponentiation button do not have accessible labels, rendering them inaccessible to screen reader users. Also, some of the scientific buttons just use their visual label, causing them to be misread (eg. "ln" is announced as "lane" instead of "L N" or "natural log").

**To Reproduce**
1. Enable TalkBack
2. Open OpenCalc
3. Tap and hold, then drag around the calculator
4. Observe that some buttons are not labeled correctly

**Expected behavior**
These buttons should be labeled accurately.

**Screenshots/videos**
(I tried recording a video, but it didn't seem to pick up TalkBack ^^;)

**Smartphone (please complete the following information):**
 - Device: Nokia XR21
 - OS: Android 13 (Play system update 2023-11-01, security update 2023-12-05)
 - App version: 3.0.0 beta 4
 - Language: English (US) systemwide, English (UK) TTS

**Comment by inson1** [comment]
I think its feature request not a bug, after its done its should be added to feature list

**Comment by inson1** [comment]
So we should add to all elements contentDescription attribute and add translations of those descriptions https://developer.android.com/guide/topics/ui/accessibility/principles#label-elements

**Comment by Darkempire78** [comment]
Open a Pull request for this please

**Comment by subhajit-rajak** [comment]
Hello!👋 @Darkempire78 I have opened a pull request for this issue. [Click here to view](https://github.com/Darkempire78/OpenCalc/pull/483)

**Comment by Darkempire78** [comment]
Done
